pkg/api/v1/types.go

```diff
@@ -2047,8 +2047,10 @@ type NodeSpec struct {
 
 // DaemonEndpoint contains information about a single Daemon endpoint.
 type DaemonEndpoint struct {
-	// Port number of the given endpoint.
-	Port int32 `json:port`
+	// Port number of the given endpoint. The tag was not properly in quotes in
+	// earlier releases, so it must be uppercased for backwards compat (since it
+	// was falling back to var name of 'Port').
```

Comment:
I'm not sure we want the comment showing up in the docs. @bgrant0607 Should he put  it in a separate comment block above this with a blank line between?